teamscale_client/client.py

```diff
@@ -50,11 +48,10 @@ def check_api_version(self):
         url = self.get_global_service_url('service-api-info')
         response = self.get(url)
         json_response = response.json()
-        self.supports_at_least_api_version_5 = int(json_response['minSupportedApiVersion']) >= 5
         api_version = json_response['apiVersion']
-        if api_version < 3:
+        if api_version < 5:
             raise ServiceError("Server api version " + str(
-                api_version) + " too low and not compatible. This client requires Teamscale 3.2 or newer.");
+                api_version) + " too low and not compatible. This client requires Teamscale 3.9 or newer.")
```

Comment:
Isn't this 4.1 or newer?

Reply:
From EServiceApiVersion.java, version 5 is from Teamscale 3.9

Reply:
We will need a new EServiceApiVersion 6, for TS 4.1